pyproject.toml

```diff
@@ -24,6 +82,5 @@ addopts = "-pno:warnings --verbose --doctest-modules --current-env --sanitize-wi
 [tool.coverage.html]
 directory = "docs/coverage"
 
-[build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2"]
-build-backend = "setuptools.build_meta"
+[template.plugins.default]
+src-layout = true

```

Comment:
This is for Hatch's global config https://hatch.pypa.io/latest/config/project-templates/

```suggestion

```